passenger-service/src/main/java/com/software/modsen/passengerservice/exception/PassengerUpdateLockException.java

```diff
@@ -0,0 +1,11 @@
+package com.software.modsen.passengerservice.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(value = HttpStatus.CONFLICT)
```

Comment:
I think that it's not "Conflict" code, but BadRequest. To be honest - it's better to check all 4XX statuses in HTTP docs and to find the most suitable. 

Not a mistake